deps/fast_float/Makefile

```diff
@@ -0,0 +1,25 @@
+# Fallback to gcc/g++ when $CC or $CXX is not in $PATH.
+CC ?= gcc
+CXX ?= g++
+
+CFLAGS=-Wall -O3
+# This avoids loosing the fastfloat specific compile flags when we override the CFLAGS via the main project
+FASTFLOAT_CFLAGS=-std=c++11 -DFASTFLOAT_ALLOWS_LEADING_PLUS
+FASTFLOAT_CFLAGS += -DFASTFLOAT_SKIP_WHITE_SPACE
```

Comment:
`FASTFLOAT_SKIP_WHITE_SPACE` is not defined in redis 
 e.g., https://github.com/redis/redis/blob/e2608478b6a46998bb3196b9b3ef4a24cbbae8a7/deps/fast_float/Makefile#L7
 
 Why do we need it?

Reply:
We need it to parse latitude in cases like this:
`HSET geo1 g '29.1, 34.2'`

In `parseGeo()` we have:
```c
  char *pos = strpbrk(str, " ,");
  ...
  *pos = '\0';
  pos++;
  ...
  *lon = fast_float_strtod(str, &end1);
  *lat = fast_float_strtod(pos, &end2);
```

Using the string of the previous example, without skipping whitespaces:
```c
  *lon = fast_float_strtod("29.1", &end1);  --> 29.1
  *lat = fast_float_strtod(" 34.2", &end2); --> 0    Error!
```